src/styles/utilitis/_fonts-weight.scss

```diff
@@ -0,0 +1,11 @@
+.text-bold {
```

Comment:
I'm not sure that you have to create separate blocks with font settings. Consider creating modifiers, extends or mixins for this purpose